python/ray/util/state/common.py

```diff
@@ -507,6 +507,9 @@ class NodeState(StateSchema):
     #: ALIVE: The node is alive.
     #: DEAD: The node is dead.
     state: TypeNodeStatus = state_column(filterable=True)
+    #: The state message of the node.
+    #: This provides more detailed information about the node's state.
+    state_message: Optional[str] = state_column(filterable=True)
```

Comment:
filterable should be false

Reply:
With filterable=True, user can for example filter all nodes that are terminated unexpectedly. Do we want to disable this?

Reply:
If we want to make it filterable, it should be enum. Currently what we have is just free-form string

Reply:
There are many string fields have `filterable=True`. Yes, the message may be somewhat free-form, I was thinking filtering on it may still be useful. But sure, let me set it to False.

Reply:
I guess a main issue is that filter only supports `=` and `!=` not substring, regex.

I think we can start with False and change to True if needed in the future